src/app/grid/grid-sample-selection-template-excel/grid-sample-selection-template-excel.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="grid__wrapper">
+    <igx-grid #myGrid [data]="data" [primaryKey]="'ID'" [rowSelection]="'multiple'" [cellSelection]="'none'"
+        [paging]="true" [height]="'530px'" width="100%">
+        <igx-column [field]="'ContactName'" [width]="'20%'"></igx-column>
+        <igx-column [field]="'Country'" [width]="'20%'"></igx-column>
+        <igx-column [field]="'City'" [width]="'20%'" dataType="number"></igx-column>
+        <igx-column [field]="'PostalCode'" [width]="'20%'" dataType="number"></igx-column>
+        <igx-column [field]="'CompanyName'" [width]="'20%'" dataType="number"></igx-column>
+
+        <ng-template igxHeadSelector let-headContext>
+            <div class="header-selector">
+                <div *ngIf="headContext.selectedCount === headContext.totalCount && headContext.totalCount !== 0"
```

Comment:
You can just use one div and [`[ngClass]`](https://angular.io/guide/template-syntax#ngclass) to swap the classes.

Reply:
Also you might be able to do this with one of the material icons: https://material.io/resources/icons/?icon=signal_cellular_4_bar&style=baseline